### PR TITLE
This commit fixes #78 issue.

### LINKED ISSUE #78: Add <, >, <=, >= matchers
We need 4 additional matchers:
be_lt / be_less_than / "<"
be_gt / be_greater_than / ">"
be_le / be_less_equal / "<="
be_ge / be_greater_equal / ">="

Those matchers should operate on following data types:
number
date family
interval family

### PR DESCRIPTION
**Pull request by proldan**
Create new matchers:
- be_less_than
- be_less_or_equal
- be_greater_than
- be_greater_or_equal

Create new expectations for intervals:
- ut_expectation_yminterval
- ut_expectation_dsinterval

### CODE REVIEW

**Comment by jgebal** [comment]
Fixes issue #76

**Comment by jgebal** [comment]
@proldan can you resolve conflicts?

File: source/core/ut_utils.pks [hunk 1]
```diff
@@ -11,21 +11,21 @@ create or replace package ut_utils authid definer is
     tr_failure - one or more asserts failed
     tr_error   - exception was raised
   */
-  tr_ignore                  constant number(1) := 0; -- test/suite was ignored
```
Comment (jgebal) [inline source/core/ut_utils.pks]: Those changes are not needed.

File: tests/sqlnet.log [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (jgebal) [inline tests/sqlnet.log]: Can you remove this file?

File: source/core/ut_utils.pkb [hunk 3]
```diff
@@ -114,6 +114,16 @@ create or replace package body ut_utils is
   begin
     return coalesce(to_char(a_value,gc_timestamp_format), gc_null_string);
   end;
+  
+  function to_string(a_value yminterval_unconstrained) return VARCHAR2 IS
```
Comment (Pazus) [inline source/core/ut_utils.pkb]: according to the naming convention we dont use uppercased keywords

**Comment by Pazus** [comment]
Great job, @proldan ! Please address minor comments and PR will be merged.

**Comment by proldan** [comment]
- Uppercase corrected
- Removed file SQLNet.log

**Comment by jgebal** [comment]
@proldan Thank you for the contribution.
Since you have added support for interval datatypes, could you add the new types for be_between matcher. This would make the PR a complete end-to-end functionality.

**Pull request merged**